src/veh_interact.cpp

```diff
@@ -1838,7 +1833,7 @@ void veh_interact::display_details( const vpart_info *part )
                    "%s: <color_ltgray>%.1f%s</color>",
                    small_mode ? _("Wgt") : _("Weight"),
                    convert_weight(item::find_type( part->item )->weight),
-                   OPTIONS["USE_METRIC_WEIGHTS"].getValue() == "lbs" ? "lb" : "kg");
```

Comment:
Confirmation that the usage of standard form of units record (```lbs```) is not an issue for this place:

![lbs](https://cloud.githubusercontent.com/assets/5421954/11644438/2ed405b6-9d5d-11e5-9073-e9d131f03b39.png)